packages/amplify-category-auth/src/index.js

```diff
@@ -312,6 +318,24 @@ async function executeAmplifyCommand(context) {
   await commandModule.run(context);
 }
 
+/**
+ * Entry point for headless commands
+ * @param {any} context The amplify context object
+ * @param {string} headlessPayload The serialized payload from the platform
+ */
+const executeAmplifyHeadlessCommand = async (context, headlessPayload) => {
+  switch (context.input.command) {
+    case 'add':
+      if (projectHasAuth(context)) return;
+      await validateAddAuthRequest(headlessPayload)
+        .then(addAuthRequestAdaptorFactory(context.amplify.getProjectConfig().frontend))
+        .then(getAddAuthHandler(context));
+      return;
+    default:
+      context.print.error(`Headless mode for ${context.input.command} auth is not implemented yet`);
```

Comment:
missing `break` this way it is a fall-through statement

Reply:
fixed